test/tls13encryptiontest.c

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/ssl.h>
+#include <openssl/evp.h>
+#include "../ssl/ssl_locl.h"
+#include "../ssl/record/record_locl.h"
+
+#include "testutil.h"
+#include "test_main.h"
+
+struct record_data {
+    const char *plaintext;
+    const char *ciphertext;
+    const char *key;
+    const char *iv;
+    const char *seq;
+} refdata[] = {
+    {
+        "0800001e001c000a00140012001d0017001800190100010101020103010400000000"
```

Comment:
that looks longer than 509 chars and will therefore break on some platforms.  and you need a comment saying how this data was generated or where it came from.